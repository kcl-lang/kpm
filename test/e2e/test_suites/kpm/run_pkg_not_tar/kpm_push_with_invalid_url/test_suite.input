kpm push invalid_url