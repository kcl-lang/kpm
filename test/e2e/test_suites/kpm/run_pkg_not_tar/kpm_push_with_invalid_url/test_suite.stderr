kpm: the current package 'kcl1' will be pushed.
kpm: oci ref is not an url.
